# std_msgs/Header
# geometry_msgs/Point tvec
# float64[9] rmat
# [ 0, 1, 2]
# [ 3, 4, 5]
# [ 6, 7, 8]

Header header
geometry_msgs/Point tvec
float64[9] rmat
